src/lib/input-parser.ts

```diff
@@ -79,14 +58,10 @@ export function getInputType(destination: string): ParsedInput {
   }
 
   // Onchain
-  if (isValidBitcoinAddress(destination)) {
-    return {
-      type: 'onchain',
-      value: destination
-    }
+  return {
+    type: 'onchain',
+    value: destination
```

Comment:
I think we might be better off using a onchain address regex check here that covers all address types. It won't validate if the address is actually valid, but will still correctly categorise it in the UI. We can then still return unknown as the final case for actual input we don't know what to do with.

Reply:
Good call, updated